src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -1,13 +1,85 @@
 import classnames from 'classnames';
 import type { ComponentChildren } from 'preact';
+import { useEffect } from 'preact/hooks';
+import {
+  Route,
+  Router,
+  Switch,
+  Link as RouteLink,
+  useLocation,
+  useRoute,
+} from 'wouter-preact';
 
 import type { PlaygroundAppProps } from '../';
 import { Link, LogoIcon } from '../../';
-import { useRoute } from '../router';
 import { componentGroups, getRoutes } from '../routes';
 import type { PlaygroundRoute } from '../routes';
 import Library from './Library';
 
+/**
```

Comment:
No meaningful changes to `NavHeader`, `NavSection` or `NavList`: just extracting them so that they are not inside of the `PlaygroundApp` component function.